specification/signalr/resource-manager/Microsoft.SignalRService/preview/2025-01-01-preview/signalr.json

```diff
@@ -102,63 +148,50 @@
     },
     "/subscriptions/{subscriptionId}/providers/Microsoft.SignalRService/locations/{location}/usages": {
       "get": {
-        "tags": [
-          "SignalR"
-        ],
-        "description": "List resource usage quotas by location.",
         "operationId": "Usages_List",
-        "produces": [
-          "application/json"
-        ],
+        "description": "List resource usage quotas by location.",
         "parameters": [
-          {
-            "in": "path",
-            "name": "location",
-            "description": "the location like \"eastus\"",
-            "required": true,
-            "type": "string"
-          },
           {
             "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
           },
           {
             "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/LocationParameter"
```

Comment:
Is this change to common type required?

Reply:
This is by using standard LocationParameter in arm lib.
```
  list is ArmProviderActionSync<
    Response = SignalRUsageList,
    Scope = SubscriptionActionScope,
    Parameters = LocationParameter
  >;
```
and Swagger version of it be https://github.com/Azure/azure-rest-api-specs/blob/main/specification/common-types/resource-management/v5/types.json#L656-L664

They should be same, except `description` and one more constraint of `minLength=1` (which is expected).